application/cms/models.py

```diff
@@ -1078,7 +1077,7 @@ class Measure(db.Model):
 
     @property
     def has_published_version(self):
-        return any(version.published for version in self.versions)
+        return any((version.status == "APPROVED" and version.published_at is not None) for version in self.versions)
```

Comment:
Is there any reason we need to check that `version.published_at is not None` here?

Reply:
Yeah I'm not sure, there are a few places where I've done this.  It feels like status APPROVED *should* be all we need?  I think I was being belt-and-braces overcautious.